ruby-challenges.rb

```diff
@@ -19,17 +24,48 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
-
+def sum_of_all array
+    array.inject() {|sum, value| sum += value} 
```

Comment:
Awesome!  Ruby also has a built-in method called sum that is a little simpler and faster on the computer.